semconv/src/main/java/io/opentelemetry/semconv/trace/attributes/SemanticAttributes.java

```diff
@@ -1111,10 +1133,6 @@ public static final class NetTransportValues {
     public static final String IP_TCP = "ip_tcp";
     /** ip_udp. */
     public static final String IP_UDP = "ip_udp";
-    /** Another IP-based protocol. */
-    public static final String IP = "ip";
-    /** Unix Domain socket. See below. */
-    public static final String UNIX = "unix";
```

Comment:
I think we could modify the generator around the lines 87-88 so that it'd detect if it's about to output `NetTransportValues` and if so, add these two deprecated values as well.

Reply:
are you ok with adding this as a followup? I'm not sure I see exactly how this would work (my jinja skills are very poor).

Reply:
never mind. I figured it out. added them as deprecated here.